recipes/libgfortran-abi/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.5.0" %}
+
+package:
+  name: libgfortran-abi
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true                    # [win]
+
+requirements:
+  run:
+    - libgfortran 3.0.0
```

Comment:
What version of gcc did you get from yum?  The ABI is very tightly tied to the particular version of gcc/gfortran.

I'll look into it on mac.  I'm sure there's a way.  I think this is where we can use "filters" or custom functions that we can define on a per-recipe basis to feed into jinja2.